spec/services/cfe/submit_assets_service_spec.rb

```diff
@@ -19,6 +67,25 @@
       end
     end
 
+    context "when there are no homes" do
+      let(:session_data) do
```

Comment:
I do see the merit, but I think in this case, at the level of a unit test I think we want the test to be explicit about what a session with no homes in it looks like. Here, the first 4 key-value pairs of the session data we've specified are very much what the test is about - I wouldn't want to relegate them to a trait. That being said, the final three lines aren't relevant to the test. They just happen to be what you would need to have valid session data to cause the SubmitAssetsService not to also do some other thing here. So maybe this would be best expressed as something like:

```ruby
let(:session_data) do
  FactoryBot.build(:session, 
                   :without_capital_assets, 
                   property_owned: "none",
                   partner_property_owned: "none",
                   property_value: "0,
                   partner_property_value: 0,
 ```
 
 That way the stuff we care about is explicit, the stuff we don't is implicit.
 
 I dunno. I think there's at least one more iteration to do on these tests. Maybe correct use of fixtures is at the heard of it.

Reply:
Yeah my choice of trait was a silly one 🤦‍♀️ but definitely! Stuff that doesn't need to be explicit could be potentially handled in a trait/a fixture. It might also help someone reading the tests to work out what kind of data needs to be set and why. That could be nice but probably needs some extra thought around the architecture for that, perhaps something for a future iteration 👍